protocols/netty/src/main/java/org/apache/james/protocols/netty/ConnectionPerIpLimitUpstreamHandler.java

```diff
@@ -46,19 +46,6 @@ public ConnectionPerIpLimitUpstreamHandler(int maxConnectionsPerIp) {
         this.maxConnectionsPerIp = maxConnectionsPerIp;
```

Comment:
`maxConnectionsPerIp` could now be declared final and the `volatil` keyword be removed